.travis.yml

```diff
@@ -14,13 +14,15 @@ env:
     # - MODE=model PR_ONLY=false
     - MODE=BreakingChange PR_ONLY=true CHECK_NAME="Breaking Changes"
     - MODE=lintdiff PR_ONLY=true CHECK_NAME="Linter Diff" NODE_OPTIONS=--max-old-space-size=8192
+    - MODE=spellcheck CHECK_NAME="Spell Checking"
```

Comment:
We are switching to Azure Dev Ops so we don't extend travis-ci

Reply:
So, undo this change, and add the step to the azure-pipelines.yml instead?

Reply:
I switched it over to the pipelines setup, but it won't preview (AFAIK this is a default security thing)